libs/shared/src/consts/handlebar-helpers/handlebarHelpers.ts

```diff
@@ -10,6 +10,12 @@ export enum HandlebarHelpersEnum {
   SORT_BY = 'sortBy',
   NUMBERFORMAT = 'numberFormat',
   I18N = 'i18n',
+  GT = 'gt',
+  GTE = 'gte',
+  LT = 'lt',
```

Comment:
There is existing handlebar helper `EQUALS` (`==`) and it might be confused with `EQ` (`===`). 

Let me know if you have better ideas for naming.